Dockerfile

```diff
@@ -23,10 +23,12 @@ RUN apk add --no-cache gcc g++
 
 WORKDIR $GOPATH/src/github.com/grafana/grafana
 
-COPY go.mod go.sum ./
+COPY go.mod go.sum embed.go ./
 
 RUN go mod verify
 
+COPY cue cue
+COPY public/app/plugins public/app/plugins
```

Comment:
Why is this directory necessary?

Reply:
From `embed.go`:

```go
//go:embed cue/*/*.cue
...
//go:embed public/app/plugins/*/*/*.cue public/app/plugins/*/*/plugin.json
```

The files get embedded during the build.

I couldn't find a way to copy _only_ cue/json files from the public/app/plugins, so I kept the whole directory tree.